code/_globalvars/lists/flavor_misc.dm

```diff
@@ -171,7 +173,7 @@ GLOBAL_LIST_INIT(TAGGERLOCATIONS, list("Disposals",
 	"Robotics", "HoP Office", "Library", "Chapel", "Theatre",
 	"Bar", "Kitchen", "Hydroponics", "Janitor Closet","Genetics",
 	"Testing Range", "Toxins", "Dormitories", "Virology",
-	"Xenobiology", "Law Office","Detective's Office"))
+	"Xenobiology", "Law Office","Detective's Office","Plumbing","Morgue"))
```

Comment:
Adding new things to this is a bad idea, otherwise you should be updating every other maps disposals to allow for this.

Reply:
Crusader told me that the parcels will just go back to the mailroom and nothing should break.

Reply:
It's best to not do this still, as it causes inconsistency with maps that don't have these defined. I can tell you a better way of implementing this if you grab me on discord.